Marksman/Server.fs

```diff
@@ -553,7 +553,16 @@ type MarksmanServer(client: MarksmanClient) =
 
             let serverCaps = ServerUtil.mkServerCaps configuredExts textSyncKind par
 
-            let initResult = { InitializeResult.Default with Capabilities = serverCaps }
+            let serverAssembly = typeof<MarksmanServer>.Assembly.GetName()
+
+            let initResult = {
```

Comment:
@billf could you follow the same pattern here of `{ InitializeResult.Default with ... }`?